packages/botonic-plugin-contentful/src/contentful/contents/button.ts

```diff
@@ -161,6 +131,7 @@ type ButtonTarget = contentful.Entry<
   | QueueFields
   | HourRangeFields
   | ScheduleFields
+  | HandoffFields
```

Comment:
DRY: ButtonTarget & CallbackTarget are too similar.
Maybe ButtonTarget can be set to initially equal CallbackTarget and in the future we can improve it?